CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# Changelog
+
+## 4.0.0
+### Breaking Changes
+- Renamed executable command from `license-checker-ci` to `d2l-license-checker`.
+- Renamed special exemption identifier from `D2L-Open-Source-Special-Exemption` to `Special-Exemption`.
+- Changed specifying a custom config file from a positional argument to `--config-file` option.
+
+### Enhancements
+- Added `--no-d2l` option to bypass built-in defaults that are D2L-specific.
+- Allow specifying `"acceptedLicenses"` in config file when `--no-d2l` is used.
```

Comment:
Again trying to make this tool be less D2L-centric when desired to entice non-D2L users.